### ISSUE TITLE
Support long duration display with hours

### ISSUE THREAD

**Issue by ashveen** [lhz516/react-h5-audio-player#59]
Duration always shows as minutes:seconds e.g. 109:18 for a long duration > 1 hour
Would be great to display it in human readable form: 1:49:18 in hh:mm:ss format.

Is that something possible with the current props?

Scenario: seminar talks typically all around 1-2 hours

**Comment by lhz516** [comment]
Currently it only supports minutes and seconds, but it's 100% reasonable to support hours as well.

I will add a prop called `timeFormat` which defaults to `"mm:ss"`, and it can be set to `"hh:mm:ss"`

Before this feature is released, you can try `customProgressBarSection` prop and overwrite `CURRENT_TIME` and `DURATION` with your own formatter.
```jsx
<Player
  src="xxx"
  ref={xxx}
  customProgressBarSection={
    [formatTime(audio.currentTime), RHAP_UI.PROGRESS_BAR, formatTime(audio.duration)]
  }
/>
```
Doc: https://static.hanzluo.com/react-h5-audio-player-storybook/index.html?path=/docs/layouts-advanced--stacked

**Comment by lhz516** [comment]
Released 3.3.0 https://github.com/lhz516/react-h5-audio-player/releases/tag/v3.3.0

Closing the issue. Feel free to continue the discussion if needed.
